code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Dispenser.dm

```diff
@@ -1,14 +1,63 @@
+/datum/reagent/acetone
+	name = "Acetone"
+	id = "acetone"
+	description = "A colorless liquid solvent used in chemical synthesis."
+	taste_description = "acid"
+	reagent_state = LIQUID
+	color = "#808080"
+	metabolism = REM * 0.2
+
+/datum/reagent/acetone/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	M.adjustToxLoss(removed * 3)
+
+/datum/reagent/acetone/touch_obj(var/obj/O)	//I copied this wholesale from ethanol and could likely be converted into a shared proc. ~Techhead
+	if(istype(O, /obj/item/weapon/paper))
+		var/obj/item/weapon/paper/paperaffected = O
+		paperaffected.clearpaper()
+		usr << "The solution dissolves the ink on the paper."
+		return
+	if(istype(O, /obj/item/weapon/book))
+		if(volume < 5)
+			return
+		if(istype(O, /obj/item/weapon/book/tome))
+			usr << "<span class='notice'>The solution does nothing. Whatever this is, it isn't normal ink.</span>"
+			return
+		var/obj/item/weapon/book/affectedbook = O
+		affectedbook.dat = null
+		usr << "<span class='notice'>The solution dissolves the ink on the book.</span>"
+	return
+
```

Comment:
Don't want this to replace anything we've already got, fine on its own, though. Also I'm pretty sure acetone is neutral, and wouldn't taste like an acid.